File: Assignment_No_14/program1.java
import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;
        int iNo = 0;
        
        for(i = 1, iNo = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++, iNo++)
            {
                if(iNo > 9)
                {
                    iNo = 1;
                }
                
                System.out.print(iNo+" ");
            }
            System.out.println();
        }
    }
}

class program1
{
    public static void main(String arr[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of rows : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}

File: Assignment_No_42/program1.cpp
#include<iostream>
using namespace std;

void Display(int iNo)
{
    if(1 <= iNo)
    {
        printf("%d * ",iNo);

        Display(--iNo);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}